LibrariesPrototype/Screens/WhisperMessage/ViewModels/WhisperMainViewModel.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  WhisperMainViewModel.swift
+//  LibrariesPrototype
+//
+//  Created by Ozgun Zor on 4/22/19.
+//  Copyright © 2019 Ozgun Zor. All rights reserved.
+//
+
+import UIKit
+
+enum WhisperMenuCase: Int {
```

Comment:
as you already know enum auto increment can be used with negative numbers as well.